app/models/rental.py

```diff
@@ -1,4 +1,57 @@
 from app import db
+from .video import Video 
+from .customer import Customer
+import datetime
 
 class Rental(db.Model):
-    id = db.Column(db.Integer, primary_key=True)
\ No newline at end of file
+    __tablename__ = "rentals"
+    id = db.Column(db.Integer, primary_key =True, nullable=False)
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.id'), nullable=False) 
+    video_id = db.Column(db.Integer, db.ForeignKey('video.id'), nullable=False) 
+    due_date = db.Column(db.DateTime, nullable=False)
+
+ #finds the number of videos that are checked out from that video id
+    def find_rentals(self, video_id):   
+        count= 0
+        rentals = Rental.query.all()
+        for rental in rentals:
+            if rental.video_id == video_id:
+                count += 1
+        return count
+
+    def get_available_inventory(self):
+        rental = Video.query.get(self.video_id)
```

Comment:
Naming this variable `rental` is a bit confusing. It might be clearer if it were named `video`.